django_reference_app/django_ref_config/settings.py

```diff
@@ -82,11 +82,10 @@
     'default': {
         'ENGINE': 'django.db.backends.postgresql',
         'NAME': 'django_reference',
-        'USER': os.getenv('POSTGRES_USER'),
+        'USER': os.getenv('POSTGRES_USER', 'postgres'),
         'PASSWORD': os.getenv('POSTGRES_PASSWORD'),
-        'HOST': os.getenv('POSTGRES_HOST'),
-        'PORT': os.getenv('POSTGRES_PORT', 5432)
-
+        'HOST': os.getenv('POSTGRES_HOST', 'postgres'),
+        'PORT': os.getenv('POSTGRES_PORT', 5432),
```

Comment:
Have just tested. This will break the apps connection to the on-cluster db.

Reply:
Do you need this part of the PR?

Reply:
Removed these vars from settings.py and added to the docker-compose file